### DISCUSSION TITLE
about mobile support

### DISCUSSION THREAD

**Discussion by amar-yaozhian** [leptos-rs/leptos#2568] [Q&A]
Will Leptos support mobile, like Dioxus?
If not, how can I use leptos on mobile?

**Comment by gbj** [comment]
You can use Leptos for a mobile app in a similar way to how you can use any web framework for a mobile app: using a webview wrapper like Capacitor, Tauri, or Cordova, or as a PWA (progressive web app).

**Reply by AlphabetsAlphabets** [reply]
Hi, @gbj is any leptos application a progress web app by default? Or is there any additional configuration that I need to make in order for it to become on?

**Reply by benwis** [reply]
It is not a progressive web app by default, you'll have to do whatever is needed to define one. A CSR app should be pretty close, I'd look at the MDN docs: https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/How_to/Create_a_standalone_app
